tensorflow_addons/layers/spectral_normalization.py

```diff
@@ -26,17 +26,21 @@ class SpectralNormalization(tf.keras.layers.Wrapper):
 
     See [Spectral Normalization for Generative Adversarial Networks](https://arxiv.org/abs/1802.05957).
 
-    ```python
-    net = SpectralNormalization(
-        tf.keras.layers.Conv2D(2, 2, activation="relu"),
-        input_shape=(32, 32, 3))(x)
-    net = SpectralNormalization(
-        tf.keras.layers.Conv2D(16, 5, activation="relu"))(net)
-    net = SpectralNormalization(
-        tf.keras.layers.Dense(120, activation="relu"))(net)
-    net = SpectralNormalization(
-        tf.keras.layers.Dense(n_classes))(net)
-    ```
+    >>> x = tf.constant( [[ 177,  164,  169,  165,  133,  134,  168,  174,  160,  165], [154,  146,  100,   54,   48,   19,    8,   6,
```

Comment:
Why the change to `>>>` for examples?

Reply:
This is for https://github.com/tensorflow/addons/issues/2066.

Reply:
@charlielito.Thanks for the review. >>> is used to change the docstring to doctest. Below is the issue
https://github.com/tensorflow/addons/issues/2066